applications/crossbar/priv/couchdb/schemas/access_lists.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_id": "access_lists",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": false,
+  "name": "ACL Entry",
+  "description": "Access Control List entries for device or account",
+  "additionalProperties": false,
```

Comment:
Crossbar generally allows client requests to append their own data to requests. If I want to store meta-data relevant to my app along side these ACL entries, the `additionalProperties:false` will bar me from doing that. I don't see a reason to restrict this, as you validate the actual properties you're interested in and unknown properties will be unused by the underlying code.